python/nammu/controller/AtfAreaController.py

```diff
@@ -25,6 +25,7 @@
 from ..view.AtfEditArea import AtfEditArea
 from ..view.SyntaxHighlighter import SyntaxHighlighter
 import TextLineNumber
+import re
 
 
 class AtfAreaController(object):
```

Comment:
Can we add to the method in line 130 the other methods of this class that are just propagating calls to `self.view.edit_area`? E.g. `clearToolTips`, `replaceSelection`, etc.